app/components/github_integration/webhooks/issues.py

```diff
@@ -126,26 +128,41 @@ async def _(event: events.IssuesReopened) -> None:
 
     @webhook.event.issues.edited
     async def _(event: events.IssuesEdited) -> None:
-        issue, changes = event.issue, event.changes
+        issue, old_issue_changes = event.issue, event.changes
```

Comment:
Why? There are no other issue changes so why is this one *old* issue changes? And the other interpretation of “changes belonging to old issue” doesn't work either because there *is* no old issue.


```suggestion
        issue, changes = event.issue, event.changes
```